src/pages/OperationsPage.js

```diff
@@ -47,10 +48,12 @@ class OperationsPage extends Component {
     }
 
     componentWillMount(){
-        this.props.getConfirmationsOnce();
-        this.props.getListCBE();
-        this.props.getPendingsOnce();
-        this.props.getLOCsOnce();
+        this.props.getConfirmationsOnce();   //  todo
+        this.props.getListCBE();   //  todo
+        this.props.getPendingsOnce();   //  todo
+        if (this.props.isLOCsNeedReload) {
```

Comment:
@Pavel-DV it's better to use _componentWillMount_ instead of _componentDidMount_.
I'll fix this and rename _ready_ to _isReady_ in settings components.